test/index.test.ts

```diff
@@ -250,6 +297,28 @@ describe('featurePolicy', () => {
     });
   });
 
+  it('can set everything all at once', async () => {
+    const features = DASHED_FEATURE_NAMES.reduce((result, feature) => ({
+      ...result,
+      [feature]: [`${feature}.example.com`],
+    }), {});
+
+    const response = await request(app(featurePolicy({ features })))
+      .get('/')
+      .expect('Hello world!');
+
+    const actualFeatures = response.get('feature-policy').split(';');
+    const actualFeaturesSet = new Set(actualFeatures);
+
+    expect(actualFeatures).toHaveLength(actualFeaturesSet.size);
+    expect(actualFeatures).toHaveLength(DASHED_FEATURE_NAMES.length);
+
+    DASHED_FEATURE_NAMES.forEach((feature) => {
+      const expectedStr = `${dashify(feature)} ${feature}.example.com`;
+      expect(actualFeaturesSet.has(expectedStr)).toBeTruthy();
+    });
+  });
+
```

Comment:
What happens if someone passes the dashed directive and the camelized directive? For example, what happens if someone passes `displayCapture` and `display-capture`?